lib/Braintree/CustomerGateway.php

```diff
@@ -102,6 +102,7 @@ public function createNoValidate($attribs = array())
     /**
      * create a customer from a TransparentRedirect operation
      *
+     * @deprecated since version 2.35.0
```

Comment:
Actually [2.3.0](https://github.com/braintree/braintree_php/commit/dfecae6e5e6dc45327d21e3be5dd1e21093648bf#diff-ba1450e872160266057a5277632b9969R113). Based on the change log for 2.3.0, I'd guess that all of the transparent redirect-related deprecations where in that version, but I'll confirm.